frontend/src/helpers/GetUserSession.js

```diff
@@ -0,0 +1,20 @@
+import axios from "axios";
+import { useExceptionHandler } from "../hooks/useExceptionHandler.jsx";
+
+const userSession = async () => {
+  try {
+    const requestOptions = {
+      method: "GET",
+      url: "/api/v1/session",
+    };
+    console.log("requestOptions", requestOptions);
+    const res = await axios(requestOptions);
+    console.log("res", res);
+    return res.data;
+  } catch (error) {
+    const handleException = useExceptionHandler();
+    handleException(error, "Error while getting session");
```

Comment:
@muhammad-ali-e Can you please refer [this code](https://github.com/Zipstack/unstract/blob/ad0ffcdb642ee94d0870f1eb3964338cd75624fc/frontend/src/components/custom-tools/tool-ide/ToolIde.jsx#L106) for handling exception?

Reply:
updated..  But I have few doubts here. We can clarify later